ios/engine/KMEI/KeymanEngine/Classes/KeymanWebViewController.swift

```diff
@@ -386,16 +386,14 @@ extension KeymanWebViewController: WKScriptMessageHandler {
       let dn = Int(fragment[dnRange.upperBound..<sRange.lowerBound])!
       let s = fragment[sRange.upperBound..<drRange.lowerBound]
       // This computes the number of requested right-deletion characters.
-      // Use it when we're ready to implement that.
-      // Our .insertText will need to be adjusted accordingly.
-      _ = Int(fragment[drRange.upperBound...])!
+      let dr = Int(fragment[drRange.upperBound...])!
```

Comment:
```suggestion
      let numCharsToRightDelete = Int(fragment[drRange.upperBound...])!
```

Perhaps could rename `numCharsToDelete` to `numCharsToLeftDelete` as well?